src/modules/extras/ExtrasConfig.js

```diff
@@ -0,0 +1,13 @@
+// @flow
```

Comment:
This file is quite weird. It has a name like it is a class but it isn't.
If we remove the hash function we can just put the other exports (which are only string constants) to the routing.